servers/rendering/rendering_device.cpp

```diff
@@ -1881,71 +1881,6 @@ uint32_t RenderingDevice::_texture_vrs_method_to_usage_bits() const {
 	}
 }
 
-Vector<uint8_t> RenderingDevice::_texture_get_data(Texture *tex, uint32_t p_layer, bool p_2d) {
```

Comment:
This was specialised for each `RenderingDeviceDriver`, as Metal can only map shared textures that are:

* 2D,
* 1 mipmap level
* 1 array length
* 1 sample
* not compressed or depth